internal/libs/protoio/writer_test.go

```diff
@@ -25,10 +24,10 @@ func aVote(t testing.TB) *types.Vote {
 		Round:     2,
 		Timestamp: stamp,
 		BlockID: types.BlockID{
-			Hash: tmhash.Sum([]byte("blockID_hash")),
+			Hash: crypto.Checksum([]byte("blockID_hash")),
```

Comment:
We do this often enough I wonder if we shouldn't provide `crypto.ChecksumString` too.

Reply:
yeah, but only in tests.